LoRaEngine/deployment.ci.aio.template.json

```diff
@@ -129,13 +147,16 @@
                 "value": "$LBS_SPI_SPEED"
               },
               "TC_URI": {
-                "value": "ws://localhost:5000"
+                "value": "wss://localhost:5001"
               },
               "RESET_PIN": {
                 "value": "$RESET_PIN"
               },
               "FIXED_STATION_EUI": {
                 "value": "$LBS_FIXED_STATION_EUI"
+              },
+              "TLS_SNI": {
+                "value": "false"
```

Comment:
If we use `localhost` as the CN for the server certificate and remove this env variable, wouldn't this work as well?